indra/newview/linux_tools/install.sh

```diff
@@ -144,6 +149,15 @@ backup_previous_installation()
     mv "$1" "$backup_dir" || die "Failed to create backup of existing installation!"
 }
 
+set_slurl_handler()
+{
+    install_dir=$1
+    prompt "Would you like to set Alchemy as your default SLurl handler? [Y/N]: "
```

Comment:
Let's add an echo before this like on line 66 please, so we present this prompt on a new line as (probably) intended